### DISCUSSION TITLE
教材是漫畫！鴻海教育基金會攜手雙北讓 AI 前進國中校園

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS4719-C/Python-Community-GS4719-C#592] [General]
**為向下扎根 AI 教育，鴻海教育基金會 2021 年出版 AI 長篇劇情漫畫《阿宅聯盟：決戰 AI 太陽王國》，今日宣布攜手台北市及新北市教育局合作，推動讓 AI 課程進入國中校園，自今年 9 月讓雙北的國中生可以在學校接觸到 AI 課程。**

現在的小朋友真好啊，國中就可以接觸AI這方面，害怕他們覺得難懂還以漫畫的方式來進行。

全文 : https://finance.technews.tw/2022/05/10/ai-middle-school/

**Comment by lx3211** [comment]
就像英文從小培養，現在開始從小培養AI素養，可惡，真的好羨慕

**Comment by handsomeniko** [comment]
AI自學很快拉

**Comment by Sazmafuji** [comment]
年紀越小先學有較大優勢，真羨慕

**Comment by d26488399** [comment]
他們從小就開始學這些東西，感覺都比我強了

**Comment by cordydeng** [comment]
透過有趣味性的方式來學習的確不會讓人排斥，但真正的成效不知道會如何，以我以前的經驗來看，很多東西都只是看過而已，沒有完整的解釋來龍去脈，其實有點難完全吸收

**Comment by henrywen547** [comment]
透過趣味性教材以及從小學習相關的概念，應該能培養更多對AI有興趣並繼續發展的人

**Comment by minniewei** [comment]
從小學習ai我覺得是現在刻不容緩的事情，感謝現在有人挺身而出做這件事

**Comment by Lai27330** [comment]
感覺從小開始就培養AI的知識觀念是一個很不錯的想法，能讓他們對於之後相關的科技發展更加敏銳，但希望編者能將內容豐富有趣的呈現，因為若是無法吸引孩童們的興趣反而可能導致學生產生排斥。

**Comment by leon10456** [comment]
感謝樓主的分享

**Comment by liangyinglly** [comment]
台灣這個科技島終於要實施科技教育了

**Comment by HsuChen88** [comment]
這麼早就開始學的話
大部分的學生應該也不知道在幹嘛吧哈哈

**Comment by etin2468** [comment]
從小就有在接觸的話，感覺對未來各個領域都很有幫助

**Comment by Lynn12378** [comment]
感覺可以激發一些孩子對於科技的興趣

**Comment by Jimmy911218** [comment]
感覺能讓小學生更親自想學ai

**Comment by yenwei12138** [comment]
好ㄟ，以前跟大人說看漫畫也可以學到東西都不信，現在要變真的了
